File: pset6/mario/mario.py
import cs50

# prompt user for the height of the pyramid. height should be between (1,8)
height = cs50.get_int("Height: ")

# asking user to enter height again if its not in between (1,8)
while (height < 1 or height > 8):
    height = cs50.get_int("Height: ")

# create patern vertically.
for i in range(1, height + 1):
    l = height - i
    for j in range(0, l):
        print(" ", end="")
    # k loop will create horizontal pressure.
    for k in range(0, 2 * i):
        if k == i:
            print("  ", end="")
        print("#", end="")
    print("\n", end="")

File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    int n, l;
    //accepting only positive integer in the range (1,8).
    do
    {
        //get positive integer from user
        n = get_int("Height: ");

    }
    while (n < 1 || n > 8);
    
    for (int i = 1; i <= n; i++) //this loop creates the pattern vertically.
    {
        l = n - i;
        for (int k = 0; k < l; k++) 
        {
            printf(" "); 
        }
        for (int j = 0; j < 2 * i; j++) //this loop creats the pattern horizontally.
        {

            if (j == i)
            {
                printf("  ");

            }
            printf("#");

        }
        printf("\n");
        
    }
}